lang/py/avro/tether/tether_task.py

```diff
@@ -339,18 +341,15 @@ def configure(self,taskType,  inSchemaText,  outSchemaText):
       estr= traceback.format_exc()
       self.fail(estr)
 
-  def set_partitions(self,npartitions):
-
-    try:
-      self._partitions=npartitions
-    except Exception as e:
-      estr= traceback.format_exc()
-      self.fail(estr)
-
-  def get_partitions():
```

Comment:
This is illegal because it is missing the `self` parameter. Mypy caught it. I converted it into a property because it is idiomatic.

Reply:
It already pays of :)